lib/rules/require-quoted-keys-in-objects.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Requires quoted keys in objects.
+ *
+ * Type: `Boolean`
+ *
+ * Values:
+ *
+ *  - `true` for strict mode
+ *
+ * #### Example
+ *
+ * ```js
+ * "requiresQuotedKeysInObjects": true
+ * ```
+ *
+ * ##### Valid for mode `true`
+ *
+ * ```js
+  * var x = { 'a': { "default": 1 } };
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * var x = { a: 1 };
+ * ```
+ */
+
+var assert = require('assert');
+var utils = require('../utils');
+
+module.exports = function() { };
+
+module.exports.prototype = {
+
+    configure: function(requireQuotedKeysInObjects) {
+        assert(
+            typeof requireQuotedKeysInObjects === 'boolean',
+            this.getOptionName() + ' option requires boolean value'
```

Comment:
Nope, i think we need interface that would accept declaration with acceptable values and do this checks automatically.

But i'm not sure if we have talked about this or not. Waiting for @mdevils to respond.